python/cudf/setup.py

```diff
@@ -21,9 +16,9 @@
     "typing_extensions",
     # Allow floating minor versions for Arrow.
     "pyarrow==10",
-    f"rmm{cuda_suffix}==23.4.*",
-    f"ptxcompiler{cuda_suffix}",
-    f"cubinlinker{cuda_suffix}",
+    "rmm==23.4.*",
+    "ptxcompiler",
+    "cubinlinker",
```

Comment:
Does this have to go along with changes in those repos (to change the wheel names?)

AKA, what does changing the names of the dependencies have to do with removing versioneer?

Reply:
That's the second half of the PR, which was added after you last reviewed. I combined these changes because it was going to add significant complexity to the CI shared workflow to be able to make the version changes and the cuda suffix changes independently. In order to use the latest workflow, each RAPIDS repo needs to make both changes at once. I agree, it's suboptimal, but the tradeoff is added complexity in the shared workflow that we (me and ops folks) decided wasn't worth it.

Reply:
Thanks, makes sense.